### PR TITLE
Introduce progressbar mixin.

### PR DESCRIPTION
**Pull request by bierik**
progressbar

### CODE REVIEW

File: ftw/theming/resources/scss/globals/helpers.scss [hunk 1]
```diff
@@ -66,8 +66,33 @@
   &:hover {
     @include auto-text-color($color);
   }
+  &:visited {
+    @include auto-text-color($color);
+    &:hover {
+      @include auto-text-color($color);
```
Comment (jone) [inline ftw/theming/resources/scss/globals/helpers.scss]: Are these changes on purpose or by accident?
Looks unrelated :wink:

File: ftw/theming/resources/scss/elements/progressbar.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
+$progressbar-animation: move;
```
Comment (jone) [inline ftw/theming/resources/scss/elements/progressbar.scss]: A `!default` would make that customizable :wink:

File: ftw/theming/resources/scss/elements/list.scss [hunk 3]
```diff
@@ -12,12 +12,16 @@ $color-list-hover: $color-gray-light !default;
...
-      display: block;
-      padding: $padding-vertical $padding-horizontal;
+      @include list-item();
     }
   }
 }
 
+@mixin list-item() {
+  display: block;
+  padding: $padding-vertical $padding-horizontal;
```
Comment (jone) [inline ftw/theming/resources/scss/elements/list.scss]: This seems to be unrelated with progress bars too. Rebase error?

**Comment by jone** [comment]
Looks nice :+1:

**Comment by bierik** [comment]
@jone updated
